README.md

```diff
@@ -1 +1,18 @@
 # hw01
+## Hello there! 
+
+My name is Julia Pruin and I am a new MAPSS student here at UChicago. I am concentrating in Psychology and working with [Dr. Monica Rosenberg](https://psychology.uchicago.edu/directory/monica-rosenberg) for my thesis. I plan to research the cognitive measures that might be used to predict changing ADHD diagnoses in an adolescent sample. 
+
+## Some Fun Facts
+
+ - I am originally from San Antonio, TX and this is my first time living outside Texas.
+ - I love to bake and cook. If I am stressed, I make cookies or bread.
+ - I am not allergic to mosquitoes. I still get bit, but I don't itch or swell.
+ - I have a pitbull. Her name is Hayley. She is a baby and I miss her very much.
+
+## Some Progress Notes
```

Comment:
Since you have a lot of text in this section, I think using more emphasis points (italics or bold, etc.) or something might make the presentation easier. Overall, I really liked the way you shared your thoughts about this process!